### ISSUE TITLE
Toast on web can be out of view

### ISSUE THREAD

**Issue by xinyumz** [GeekyAnts/NativeBase#4975]
### Description

Toast **on web** is showing at the top/bottom of the page - can be out of view and have to scroll all the way up/down, it would be nice if it can be constrained to the top/bottom within the view. 
The todo-list example in the docs has this problem too. If you add enough todos, the toast will be out of view and you'll need to scroll to see it. 

### Problem Statement

Toast is a very commonly used component, the problem described can happen easily even with given example in the docs. 

### Proposed Solution or API

N/A

**Comment by Viraj-10** [comment]
Hi @xinyumz, Thanks for reporting the issue. we will have a look.

**Comment by xinyumz** [comment]
I did figure out a way to fix it. For the web version, define the dimensions to window size:

```
import { Dimensions, Platform } from "react-native";
import { NativeBaseProvider, ScrollView, ... } from "native-base";


//inside the component, define dimensions
const windowWidth = Dimensions.get("window").width;
const windowHeight = Dimensions.get("window").height;

//set height and width, for example:
<NativeBaseProvider>
   <ScrollView
       h={Platform.OS === "web" ? webHeight : undefined}
       w={Platform.OS === "web" ? webWidth : undefined}
     >
         ... 
   </ScrollView>
</NativeBaseProvider>
```

This way, adding toasts, modals and etc, will be confined to the screen instead of being placed on top of the entire component. 
I'm also open to any better ways to do this!

**Comment by Viraj-10** [comment]
Hi @xinyumz, Thanks for sharing your solution. We will try to fix our toast component.

**Comment by Viraj-10** [comment]
This issue has been fixed in `v3.4.8`.
